amr-wind/wind_energy/ABLBoundaryPlane.H

```diff
@@ -176,16 +194,28 @@ private:
 
     //! NetCDF input level time
     amrex::Vector<amrex::Real> m_in_times;
+    amrex::Vector<int> m_in_timesteps;
 
     //! Inlet data
     InletData m_in_data;
-#endif
 
     //! IO mode
     io_mode m_io_mode{io_mode::undefined};
 
     //! Flag indicating if this capability is available
     bool m_is_initialized{false};
+
+    //! controls extents on native bndry output
+    const int m_in_rad = 1;
+    const int m_out_rad = 1;
+    const int m_extent_rad = 0;
+
+    //! output format for bndry output
+#ifdef AMR_WIND_USE_NETCDF
```

Comment:
so if we turn on netcdf, then by default we would use netcdf output But if we find that the native is much faster, then we should probably default to `native`, right? Even if netcdf is turned on, e.g. for other reasons

Reply:
yep that works for me, I can change the current reg tests to use bndry_output_format=netcdf. I should probably add new ones too for native